plugins/chain_plugin/chain_plugin.cpp

```diff
@@ -2293,17 +2296,19 @@ read_only::get_table_rows_result read_only::get_kv_table_rows_context( const rea
 
          row_value.clear();
          row_value.resize(value_size);
-         status = itr->kv_it_value(offset, row_value.data(), key_size, value_size);
-         EOS_ASSERT(status == chain::kv_it_stat::iterator_ok, chain::contract_table_query_exception,  "Invalid key in ${t} ${i}", ("t", p.table)("i", p.index_name));
+         status = itr->kv_it_value(offset, row_value.data(), value_size, actual_size);
```

Comment:
This is the fix:  change key_size to value_size. It caused row_value trimmed and failed the subsequence search in line 2305.

Reply:
Good. All tests passed?